### PR TITLE
OOP school library: Add Basic UI.

### PR DESCRIPTION
**Pull request by mailsg**
Hi,

This PR is raised to demonstrate the addition of basic UI. Below are the pointers implemented:
- Created a `app.rb` file that serves as console app entry-point. It has methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.

- In the `main.rb` defined the entry point, this will be a method called `main` that is invoked at the end of file. This method does the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

Kindly review the PR and provide feedback on the same.

Thanks.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    case permission
+    when 'N'
+      student = Student.new(age, 'classroom', name: name, parent_permission: false)
+      @people << student
+    when 'Y'
+      student = Student.new(age, 'classroom', name: name, parent_permission: true)
+      @people << student
+    end
+    puts 'Student created successfully'
+  end
```
Comment (codepantha) [inline app.rb:55]: - [ ] A problem occurs when attempting to create either a `Teacher` or a `Student` as you can see below 👇 Please try to fix this bug 🐛 
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Enter the Author name: '
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books << book
+    puts 'Book created successfully'
+  end
```
Comment (codepantha) [inline app.rb:77]: - [ ] An error occurs when trying to create a new book due to not requiring the `Book` class at the top of the `app.rb` file. Please fix this issue by requiring the `book.rb` file at the top of the `app.rb` using `require_relative` 👍 
image

**NB** **The next reviewer will test the create rentals functionality and other features that couldn't be tested as a result of the highlighted issues in this review 😎**

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @mailsg 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Added basic console UI to the app 📌 
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (DeliceLydia) [reply app.rb:77]: Addressed

File: app.rb (hunk 1 above)
Comment (DeliceLydia) [reply app.rb:55]: Addressed

**Review by DeliceLydia — APPROVED** [review]
## Hi @mailsg, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes requested



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**